schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+                },
+                "connectionType": {
+                    "description": "Type of connection to device",
+                    "type": "string",
+                    "enum": ["local", "cloud"]
+                },
+                "compact": {
+                    "description": "If compact mode supported",
+                    "type": "boolean"
+                },
+                "dataFolder": {
+                    "description": "Folder relative to iobroker-data where the adapter stores the data. This folder will be backed up and restored automatically. You can use variable '%INSTANCE%' in it",
+                    "type": "string"
+                },
+                "dataSource": {
+                    "description": "How the data will be received from device",
+                    "type": "string",
+                    "enum": ["poll", "push", "assumption"]
+                },
+                "dependencies": {
+                    "description": "Array like [{\"js-controller\": \">=2.0.0\"}] that describes which ioBroker modules are required for this adapter",
+                    "type": "array",
+                    "items": {
+                        "type": ["object", "string"]
+                    }
+                },
+                "enabled": {
+                    "description": "Value should be false so new instances are disabled by default",
+                    "type": "boolean"
+                },
+                "eraseOnUpload": {
+                    "description": "erase all previous data in the directory before upload",
+                    "type": "boolean"
+                },
+                "extIcon": {
+                    "description": "Link to external icon for uninstalled adapters. Normally on GitHub",
+                    "type": "string"
+                },
+                "getHistory": {
+                    "description": "if adapter supports getHistory message",
+                    "type": "boolean"
+                },
+                "icon": {
+                    "description": "name of the local icon (should be located in subdirectory \"admin\")",
+                    "type": "string"
+                },
+                "keywords": {
+                    "description": "Similar to keywords in package.json, but can be defined in many languages.",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "localLinks": {
+                    "description": "Link to the web service of this adapter. E.g to http://localhost:5984/_utils for button in admin",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "localLink": {
+                    "description": "[DEPRECATED] Use common.localLinks",
+                    "type": "string"
+                },
```

Comment:
Should we even document deprecated properties here?

Reply:
Good point, which i was also thinking about, unfortunately, it seems like the deprecated flag is not part of json schema yet, however there is already a merged PR that introduces this feature. But if everyone agrees I would also not document them, to "force" devs to remove them.

Reply:
@Apollon77 opinion?

Reply:
see it the same: do not document them, if people ask we should remember why it is not in ... maybe having a list of "deprecated" stuff somewhere?